index.html

```diff
@@ -0,0 +1,521 @@
+                Quos sunt ad dolore ullam qui. Enim et quisquam dicta molestias.
+                Corrupti quo voluptatum eligendi autem labore.
+              </p>
+              <div class="flex space-x-8">
+                <span>£80,000</span>
+                <span>Manchester, UK</span>
+              </div>
+            </div>
+            <div class="flex justify-end items-center mt-4 space-x-2">
+              <button class="text-travel-orange text-base font-semibold">
+                View more posts
+              </button>
+              <span class="text-travel-orange">&rarr;</span>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- Testimonial -->
+      <section class="relative z-10 mt-16">
+        <div class="max-w-7xl mx-auto absolute inset-0">
+          <img
+            src="assets/images/team-photo 1.svg"
+            alt=""
+            class="w-full h-full object-cover rounded-xl"
+          />
+        </div>
+        <div
+          class="max-w-7xl mx-auto flex flex-col items-center gap-x-8 gap-y-10 px-6 md:gap-y-8 lg:px-8 xl:flex-row xl:items-stretch"
+        >
+          <div class="-mt-8 w-full max-w-2xl xl:-mb-8 xl:w-96 xl:flex-none">
+            <div
+              class="relative aspect-[2/1] h-full md:-mx-8 xl:mx-0 xl:aspect-auto"
+            >
+              <img
+                src="/assets/images/karsten-winegeart-PJYWbsSSJa8-unsplash.jpg"
+                alt="Team photo"
+                class="absolute inset-0 h-full w-full rounded-2xl object-cover shadow-2xl"
+              />
+            </div>
+          </div>
+          <div class="pl-6">
+            <figure class="relative isolate py-12 text-white">
+              <blockquote class="text-base">
+                <p>
+                  "Discovering this fitness platform has been a game-changer for
+                  me! The premium quality equipment, performance clothing, and
+                  essential supplements have transformed my workouts. The
+                  diverse range of top-notch fitness gear has not only elevated
+                  my training sessions but also made my fitness journey more
+                  enjoyable. The knowledgeable and supportive community is an
+                  added bonus. Whether you're a seasoned athlete or just
+                  starting, this platform provides everything you need for a
+                  successful fitness journey. I highly recommend it to anyone
+                  committed to achieving their health and fitness goals!"
+                </p>
+              </blockquote>
+              <div class="flex mt-6 space-x-4 text-sm">
+                <span>Judith Black</span>
+                <span>CEO of Tuple</span>
+              </div>
+            </figure>
+          </div>
+        </div>
+      </section>
+      <!-- CTA Banner section-->
+      <section class="max-w-screen-2xl mx-auto mt-32 relative">
+        <a href="#">
```

Comment:
Intentional design choice to have the entire CTA clickable, or should we just have the button clickable?
Missing link to shop page

Reply:
Intentional choice to have whole as clickable, can complete